ios/KMMIC.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -1,5 +1,14 @@
 {
   "pins" : [
+    {
+      "identity" : "nuke",
+      "kind" : "remoteSourceControl",
+      "location" : "https://github.com/kean/Nuke",
```

Comment:
What is the purpose of Nuke here? I believe this is unrelated to this PR since Nuke is for loading images.

Reply:
@phongvhd93 reverted it at [fd6c604](https://github.com/markgravity/kmm-ic/pull/75/commits/fd6c6044dbaf21e6dadd02ed81aa46b248f39f3f) 🙏